packages/create-redwood-app/tests/template.test.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-env node, es2022 */
+
+import assert from 'node:assert'
+import fs from 'node:fs'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+import { test } from 'node:test'
+
+const TEMPLATE_PATH = path.resolve(
+  path.dirname(fileURLToPath(import.meta.url)),
+  '../template'
+)
+
+// If you add, move, or remove a file from the create-redwood-app template,
+// you'll have to update this test.
```

Comment:
I thought snapshots were added to Node's test runner (or assert module), but I couldn't find them, so maybe I misread or they've been pulled out since

Reply:
Seems like they were added here, but I can't find any docs on them: https://github.com/nodejs/node/pull/44095

Reply:
This comment here hints that it was removed, but still can't find the commit: https://github.com/nodejs/node/pull/47392#issue-1652845363